### PR TITLE
Add PKGBUILDs for jupyter

### PR DESCRIPTION
**Pull request by patstew**

### CODE REVIEW

File: mingw-w64-python-jsonschema/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,59 @@
+pkgbase=python-jsonschema
```
Comment (Alexpux) [inline mingw-w64-python-jsonschema/PKGBUILD]: Fix all these lines. See proper pkgbase in first file in this commit.

Comment (Alexpux) [inline mingw-w64-python-jsonschema/PKGBUILD]: Add info about maintainer of this packages.

File: mingw-w64-python-jsonschema/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+pkgname=("${MINGW_PACKAGE_PREFIX}-python2-jsonschema" "${MINGW_PACKAGE_PREFIX}-python3-jsonschema")
+pkgver=2.5.1
+pkgrel=1
+pkgdesc="An implementation of JSON Schema validation for Python"
```
Comment (Alexpux) [inline mingw-w64-python-jsonschema/PKGBUILD]: For mingw packages we have (mingw-w64) at the end of description. Fix all packages.

File: mingw-w64-python-qtconsole/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,70 @@
+pkgbase=python-qtconsole
+pkgname=("${MINGW_PACKAGE_PREFIX}-python2-qtconsole" "${MINGW_PACKAGE_PREFIX}-python3-qtconsole")
+pkgver=4.2.1
+pkgrel=1
+pkgdesc="A language-agnostic web-based interactive shell/notebook server."
```
Comment (Alexpux) [inline mingw-w64-python-qtconsole/PKGBUILD]: I see the same description for most packages in this commit. Fix descriptions.

**Comment by Alexpux** [comment]
See my comments

**Comment by patstew** [comment]
I've addressed your comments, but the CI builds are failing in different places, and I'm not sure why in either case.

**Pull request merged**